src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -438,7 +438,7 @@ public ColumnFamilyStore(Keyspace keyspace,
 
         // Create Memtable only on online
         Memtable initialMemtable = null;
-        if (DatabaseDescriptor.isDaemonInitialized())
+        if (DatabaseDescriptor.enableMemtableInitialization())
             initialMemtable = createMemtable(new AtomicReference<>(CommitLog.instance.getCurrentPosition()));
         data = new Tracker(this, initialMemtable, loadSSTables);
```

Comment:
In DSE, if memtable is not enabled, `loadSSTables` is `false`

Reply:
Yes, I missed that